### DISCUSSION TITLE
IKEA W2049 can't turn off simulated brightness

### DISCUSSION THREAD

**Discussion by hyperion007** [Koenkk/zigbee2mqtt#7314] [Devices]
I think there is something that isn't working with setting "simulated_brightness" on the IKEA W2049 remote control. 
false or true, it still publishes the same actions: brightness_move_up, brightness_stop etc.

Also, when I press and hold the arrow buttons it first publishes:
May 03 19:43:29 payload '{"action":"on","action_group":901,"battery":100,"brightness":0,"linkquality":132,"update":{"state":"idle"},"update_available":false}'
Then:
May 03 19:43:30 payload '{"action":"on","battery":100,"brightness":0,"linkquality":132,"update":{"state":"idle"},"update_available":false}'
Then:
May 03 19:43:31 payload '{"action":"arrow_right_hold","battery":100,"brightness":0,"linkquality":132,"update":{"state":"idle"},"update_available":false}'

I can't find any information about this action group? And also, why does it publish "on" two times before "arrow_right_hold"

The delay is 2 seconds from when the remote realizes that I have pressed and held the arrow key until it publishes this.

Hope this can be solved :) 

Thank you very much!

**Comment by hyperion007** [comment]
Would be great to get some help with this issue

**Reply by MattWestb** [reply]
>I can't find any information about this action group? And also, why does it publish "on" two times before "arrow_right_hold"

You is triggering reset of the light group that is being sent to the light for getting them in sync with brightness and color.
All light command is by Zigbee standard sent to gropes of lights and not to one command to each device and there the group is coming in place for the coordinator listening on for getting the commands.

Its one light controller not one "multi click" controller.
